src/modules/deal-investor/deal-investor.service.get-investors.test.ts

```diff
@@ -0,0 +1,74 @@
+import { AcbsAuthenticationService } from '@ukef/modules/acbs/acbs-authentication.service';
+import { AcbsDealPartyService } from '@ukef/modules/acbs/acbs-deal-party.service';
+import { DealInvestorGenerator } from '@ukef-test/support/generator/deal-investor-generator';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import { when } from 'jest-when';
+
+import { DealInvestorService } from './deal-investor.service';
+
+jest.mock('@ukef/modules/acbs/acbs-deal-party.service');
+jest.mock('@ukef/modules/acbs/acbs-authentication.service');
+
+describe('DealInvestorService', () => {
+  const valueGenerator = new RandomValueGenerator();
+  const idToken = valueGenerator.string();
+  // TODO: replace with value from default configuration, or remove field portfolioIdentifier completely.
+  const portfolioIdentifier = 'E1';
+
+  let acbsAuthenticationService: AcbsAuthenticationService;
+  let acbsDealPartyService: AcbsDealPartyService;
+  let service: DealInvestorService;
+
+  let acbsDealPartyServiceGetDealPartyForDeal: jest.Mock;
+
+  beforeEach(() => {
+    acbsDealPartyService = new AcbsDealPartyService(null, null);
+
+    acbsDealPartyServiceGetDealPartyForDeal = jest.fn();
+    acbsDealPartyService.getDealPartyForDeal = acbsDealPartyServiceGetDealPartyForDeal;
+
+    acbsAuthenticationService = new AcbsAuthenticationService(null, null, null);
+    const acbsAuthenticationServiceGetIdToken = jest.fn();
+    acbsAuthenticationService.getIdToken = acbsAuthenticationServiceGetIdToken;
+
+    service = new DealInvestorService(acbsAuthenticationService, acbsDealPartyService);
+
+    when(acbsAuthenticationServiceGetIdToken).calledWith().mockResolvedValueOnce(idToken);
+  });
+
+  describe('getDealInvestors', () => {
+    const dealIdentifier = valueGenerator.ukefId();
+
+    it('returns a transformation of the deal investors', async () => {
+      const { dealInvestorsInAcbs, dealInvestors } = new DealInvestorGenerator(valueGenerator).generate({
```

Comment:
Again see comment on `deal-investor-generator.ts`, could this be changed to `const { dealInvestorsInAcbs, dealInvestorsFromService }`?

Reply:
Changed